README.md

```diff
@@ -80,6 +99,18 @@ steps:
 
   The tag that was found is made available as the step's output.
 
+* **`newer`** (`${{ steps.some_step_id.outputs.newer }}`)
+
+  If the value provied by `compares-to-latest` is newer (greater) than the latest `tag` found then `newer` output value is set to `true` otherwise `false`. Following the same pattern we also get `older` and `equal` set to `true/false`.
```

Comment:
Hmm for some reason to me it would make more sense if we were comparing the latest *to* this value.

Consider this example that I imagine: right now the *latest* is *older* than the specified value, but we are waiting for it to become *newer*. Which seems to be the opposite from how this works now.

But I will gladly hear out your opinion as well.

Reply:
Hey. Well, it might seem like this and it's been first like you suggest in the begging. But the meaning here is slightly different, have a look at the working configuration which I already use:

```yaml
      -
        uses: dysnix/find-latest-tag@v1
        id: latestTag
        with:
          compares-to-latest: ${{ github.ref }}
          repository: ${{ github.repository }}
      -
        name: Docker meta
        id: meta
        uses: docker/metadata-action@v3
        with:
          images: ${{ needs.pre.outputs.image }}
          flavor: |
            latest=${{ steps.latestTag.outputs.older == 'false' }}
            suffix=${{ matrix.suffix }}
          tags: |
            type=semver,pattern=v{{version}}
            type=semver,pattern=v{{major}}.{{minor}}
```

So read it as follows: `{provided-value} comares-to {found-latest}`, i.e. compares-to-latest. In the above example, I make a new release (tag) which means that `equal=true` (`newer=true` could also be, say it if we force overwrite tag). So checking like above gives me guaranty that I perform the most latest tag release and specify `latest` for the docker image.

Reply:
First the field was named `compares-to` and was the opposite to `compares-to-latest`, and it happened the first variant was inconvenient for my usecase.

Reply:
@oprypin Howdy. Yes totally makes more sense. Now the given repos' latest is compared to a value.